static/js/place_overview.js

```diff
@@ -47,21 +54,21 @@ function adjustMenuPosition() {
  * @param {string} dcid
  */
 function getChildPlaces(dcid) {
-  return axios.get(`/api/child-place/${dcid}`).then((resp) => {
+  return axios.get(`/api/place/child/${dcid}`).then((resp) => {
     let places = resp.data;
-    let result = [];
-    for (let dcid of places) {
-      if (!dcid.startsWith("geoId/")) {
-        // Zipcode and school district.
-        continue;
-      }
-      if (dcid.replace("geoId/", "").length > 7) {
-        // Census Tract.
-        continue;
-      }
-      result.push(dcid);
-      if (result.length == 5) {
-        break;
+    let result = {};
+    for (let place of places) {
+      for (let placeType of place["types"]) {
```

Comment:
Does this munging belong in the server API? Esp. since this app is the only user of the API, we can simplify the code here (and greatly reduce the payload size)

Reply:
Moved the logic to server side